### PR TITLE
Add appveyor config file.

### PR DESCRIPTION
**Pull request by skl131313**
Make testing a bit easier, seems dlang has their own tester for windows and don't use appveyor. But their testing is only for their own use.

It [pulls](https://github.com/Ingrater/dmd/pull/6/commits/6b0595582d64d4470f336c225a40ed891d2973dc#diff-180360612c6b8c4ed830919bbb4dd459R54) in druntime/phobos from the same branch as DMD is being tested on. So you would need to make sure the branches match.

Also building DMD with the win64.mak file is broken. It's some really odd setup that uses a separate D program in place of the DMC compiler with the same win32.mak file. That translates the switches from DMC to MSVC. It looks like the build is broken on dlang's master as well for it.

### CODE REVIEW

File: test/Makefile [hunk 1]
```diff
@@ -103,7 +103,7 @@ export SEP=$(subst /,\,/)
 DRUNTIME_PATH=..\..\druntime
 PHOBOS_PATH=..\..\phobos
 export DFLAGS=-I$(DRUNTIME_PATH)\import -I$(PHOBOS_PATH)
-export LIB=$(PHOBOS_PATH)
+#export LIB=$(PHOBOS_PATH)
```
Comment (Ingrater) [inline test/Makefile:106]: is this change intentional? I don't think we should change such a central place of the test makefile.

**Comment by Ingrater** [comment]
Sorry, but what is the point of this? If you want to add a new autotester to dmd you should do a PR against the master repository. Adding support for appveyor has nothing to do with Dll support and thus should not be part of my dll support PR.

**Comment by skl131313** [comment]
It'd be so you can run your tests. You'd remove it before creating a pull request with dlang/master.

File: test/Makefile (hunk 1 above)
Comment (skl131313) [reply test/Makefile:106]: It overwrites the LIB environment variable, some of the tests have the flag "-conf=" set to nothing. So if this line is allowed the tests won't compile as they won't know where the libraries like shell32.lib are.

**Comment by Ingrater** [comment]
> It'd be so you can run your tests. You'd remove it before creating a pull request with dlang/master.

You can do 'make LIB=something' if you want to override the value of LIB. No need to remove it from the makefile. I for example do ' make LIB="$LIB" '.
